system-tests/debug-logging-default-test/build.gradle.kts

```diff
@@ -1,7 +1,15 @@
 plugins {
-    com.teamscale.`system-test-convention`
+	com.teamscale.`system-test-convention`
 }
 
 tasks.test {
-    teamscaleAgent(mapOf("debug" to "true"))
+	teamscaleAgent(mapOf("debug" to "true"))
+
+	doFirst {
+		// the test checks if a log file is written, so we need to remove all leftover agent temp directories
+		// to avoid false-positive test failures
+		val leftoverAgentTempDirectories = file(System.getProperty("java.io.tmpdir")).listFiles()
+			.filter { it.name.contains("teamscale-java-profiler") }
```

Comment:
The system tests are running in parallel. Doesn't this become a problem if another system test writes stuff to there?

Reply:
ah, good point. Didn't think about this. I now changed the test to not use `/tmp` but a private directory as the system temp dir